test/integration/wallet.test.js

```diff
@@ -342,6 +362,7 @@ describe('@integration: Wallet', () => {
       step('Then I should see 50 rows', () => helper.shouldSeeCountInstancesOf(50, 'TransactionRow'));
       step('When I scroll to the bottom of "transactions box"', () => { wrapper.find('Waypoint').props().onEnter(); });
       step('Then I should see 75 rows', () => { wrapper.update(); helper.shouldSeeCountInstancesOf(75, 'TransactionRow'); });
+      step('When I click on a transaction row, I should be redirected to transactoinDetails step', () => helper.clickOnTransaction());
```

Comment:
This step is really two steps in GIven-When-Then.
You can then reuse this for the first step:
https://github.com/LiskHQ/lisk-hub/blob/9fc15e1ca7875c264fe69c869e83e67674c9a841/test/utils/genericStepDefinition.js#L13-L15
```
step('When I click on a transaction row', () => helper. clickOnElement('.transactions-row'));
step('Then I should be redirected to transactoinDetails step', () => helper.shouldRedirectToTransactionDetail());
```